packages/vuetify/src/components/VNavigationDrawer/VNavigationDrawer.ts

```diff
@@ -351,6 +351,8 @@ export default baseMixins.extend({
     },
     genListeners () {
       const on: Record<string, (e: Event) => void> = {
+        mouseenter: () => (this.isMouseover = true),
+        mouseleave: () => (this.isMouseover = false),
```

Comment:
Is it possible that these two lines should be enclosed in `if (this.expandOnHover) {`?